platform/ios/Integration Tests/Annotation Tests/MGLAnnotationViewIntegrationTests.m

```diff
@@ -0,0 +1,68 @@
+//
+//  MGLAnnotationViewIntegrationTests.m
```

Comment:
Oops this header snuck into the test file.

Reply:
This is WIP - but good catch :) I need to turn that off